### PR TITLE
Teacher recruitment announcement

### PR DESCRIPTION
**Pull request by tanyaparker**
image

### CODE REVIEW

**Comment by tanyaparker** [comment]
@breville just look at the first commit to ignore the i18n files. And I also made the string addition to I18n gsheet.

File: dashboard/app/views/api/teacher_announcement.html.haml [hunk 1]
```diff
@@ -5,18 +5,42 @@
...
+          %button.primary= t('home.announcement_go_beyond_button')
   .mini-announcement
     .content-text
       %i.fa.fa-hand-o-right
       != t('home.announcement_all', teacher_blog_url: teacher_blog_url)
 
 #clearboth{style: 'clear: both;'}
+
+:javascript
+  $( document ).ready(function() {
```
Comment (breville) [inline dashboard/app/views/api/teacher_announcement.html.haml]: nit: could make spacing inside ()s consistent.

File: dashboard/config/locales/en.yml [hunk 2]
```diff
@@ -390,6 +390,7 @@ en:
...
     announcement_how_code_studio: "You don't need any experience to start teaching computer science in your classroom. Code.org offers curriculum, lesson plans, high quality professional learning programs, and tons of great tools for all grade levels&mdash;everything you need, all at no cost. Wondering where to get started? Download <a href='%{video_download_url}'>this video</a>!"
     announcement_go_beyond: "Go beyond an Hour of Code and explore computer science concepts with your students every week. Code.org offers curriculum, lesson plans, high quality professional learning programs, and tons of great tools for all grade levels&mdash;and it's all free. :) You don't need any experience to start teaching computer science in your classroom. <b>Find the next step that's right for your classroom.</b>"
+    announcement_teacher_recruitment: "Teachers like you are leading the computer science education movement. You can participate in high-quality K-12th grade professional learning programs <strong>at absolutely no cost to you or your school</strong>. Take the lead on computer science in your school, and apply for professional learning."
```
Comment (breville) [inline dashboard/config/locales/en.yml:393]: "K-12th grade" read a little awkwardly until I fully parsed it.

**Comment by breville** [comment]
Interesting technique.  Would be good to work out how to share content & code between sites.

**Comment by tanyaparker** [comment]
I could put it all on dashboard and use ajax on teacher dashboard like we did with the list of courses on teacher dashboard.

**Comment by breville** [comment]
For something so temporary and small I'm not sure you need to invest more effort.

File: dashboard/app/views/api/teacher_announcement.html.haml (hunk 1 above)
Comment (tanyaparker) [reply dashboard/app/views/api/teacher_announcement.html.haml]: Oh weird, did hound not warn me because it's not a .js file? 😁

**Pull request merged**